### DISCUSSION TITLE
简单小狗 | /posts/small-dog/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#636] [Announcements]
https://yuanfan.rbind.io/posts/small-dog/

**Comment by hongtaoh** [comment]
我好像小学毕业之后再没玩过游戏。

**Reply by earfanfan** [reply]
嗯，郝糊涂，你可以数一数自己从小到大哭过的经历，我猜一定没有被游戏虐哭的时候。

**Reply by hongtaoh** [reply]
但是有被代码虐哭的时候

**Reply by earfanfan** [reply]
天啦噜，原来你也会被代码虐哭嘛！啊，我忽然觉得好感动。不过悄悄吐个槽，我被代码虐哭一般都是默默流眼泪，然后继续死磕，实在是死磕不出来就只好放着，心里却一直记着。但是被游戏虐哭，通常是嚎啕大哭，因为觉得自己无论如何都过不去那关……

**Reply by hongtaoh** [reply]
我这里说的“虐哭”是比喻，但是没有真的哭出来，只是很沮丧，特别是看别人代码又没有注释的时候。。

**Comment by CyrusYip** [comment]
我觉得能哭出来还挺健康的，至少没转换成生气。我玩空洞骑士卡关的时候（第二次遇到大黄蜂），从难过变到生气，气得我快要摔手柄。

![5743bab98ce844f29c2a5a8ec547cec2](https://github.com/user-attachments/assets/ea32bb9c-2a65-4771-bdeb-3a8b5d9a4607)


弹反我也不会，不过我玩游戏卡关就会去学习一些视频教程（有时候脑子觉得会了，手跟不上，哈哈），你也可以试试。

积木的分级（6+）应该不是说 6 岁的人能玩懂，而是说它对 6 岁和以上的人没有危害，6 岁以下的小孩可能会拿来吃。

**Reply by earfanfan** [reply]
>有时候脑子觉得会了，手跟不上

我也是卡关会看视频教程，我之前玩空洞骑士找的是 B 站上一位叫做猫诺的女孩出的教程，这次玩偏差者也是看她的教程，然而我发现她这几年真得变得好厉害，我看明白了，距离练会还有很长很长很长很长的距离。加上许久没玩，真得生疏太多了。

我玩游戏也有生气的时候，就是玩那种逃生关卡，失之毫厘、功败垂成的时候，也是气的想把手柄掰断算逑。

原来积木的分级还可以这样理解啊，国内的家长一向视游戏为猛虎，对待孩子玩游戏是较为苛责的心态，不知怎么地我就自然而然以为（6+）应该是容易的了（PS家长们对小孩子玩益智游戏是较为包容的心态，大概因此以为益智游戏应该是容易的吧）。说实话，我拼完以后，真心觉得那个积木说不定都是给小孩买回家，但是是大人才能坚持拼完的。

**Reply by earfanfan** [reply]
话说，叶寻同学你放的这个土拨鼠大叫的动图是循环播放的，我一不小心就看了好多遍，哈哈哈。然后我想起来一张梗图（也称为memes）（又PS大家都是怎么从图库里快速找到自己想要拿出来的那张图的？），大概就是在一片绿草地上有一条笔直的路，然而这条路却被放置了一个与这条路同样宽度的栏杆嘞，一个人类小伙子面对这个栏杆是跨也跨不过去、挤也挤不进去，最后哭泣着躺倒在栏杆一边。这个图配的文字通常是，我与我想象出来的困难。

**Reply by CyrusYip** [reply]
哈哈，土拨鼠的图我也很喜欢。我最近开发 [hugo-theme-yue](https://github.com/CyrusYip/hugo-theme-yue) 遇到一些 bug，搜索半天解决不了，在家就是吼「啊」。

积木的分级我没了解过，我是按游戏分级来理解的。我记得果冻有针对小孩的提示，好像说不要整个吞咽，要咬开。

我没有收集迷因图的习惯（只保存一些自己做的），一般就是需要用就搜，比如：「土拨鼠 啊」。你说这个迷因图我没见过。不知道有没有可以用文字搜迷因图的网站或者应用。
  
如果你有保存好的迷因图，可以用一些带 AI 功能的图库软件，以文字搜图。

**Reply by residualsun1** [reply]
> 大概就是在一片绿草地上有一条笔直的路，然而这条路却被放置了一个与这条路同样宽度的栏杆嘞，一个人类小伙子面对这个栏杆是跨也跨不过去、挤也挤不进去，最后哭泣着躺倒在栏杆一边。这个图配的文字通常是，我与我想象出来的困难。

![image](https://github.com/user-attachments/assets/0c60f97c-3f7b-407c-b944-bb92b06c0db8)

我之前在豆瓣上看到的这张图，一眼看过去还挺喜欢的。

我之前还看过土拨鼠的视频，土拨鼠的吼叫好像被人类的声音代替，我一开始还真以为那就是土拨鼠的叫声，看起来憨憨的土拨鼠配上吼叫声，还挺魔幻的。

**Reply by earfanfan** [reply]
@residualsun1  就是这张图。哈，黄同学你回来啦，我看到你耕了新博客，之前你说会离开很久，我还以为是很久呢，原来就是放了个暑假啊。

那个土拨鼠叫我也听过，最开始我也以为那就是土拨鼠的声音，好久以后才从 B 站弹幕上得知那是人类配音的，然而我脑子里看到那个土拨鼠叫的动图所想象的声音已经和人类的声音固定住了……

**Reply by residualsun1** [reply]
@earfanfan 是的！其实原来预期上想更久一些，但是后来我又觉得自己每个当下经历的事情以及产生的情绪都很重要，如果没能记录下来的话，就会觉得很遗憾。虽然我之前经常嫌弃自己写的东西，但还是逐渐接受了自己的一部分。

之前见过叶寻提到自己的写作习惯是慢慢修改再发布，但我后来有了别的想法，觉得自己更喜欢抓住当下的情绪，然后就记录下来。

原来你也一度以为那就是土拨鼠的声音😆。

**Reply by earfanfan** [reply]
@residualsun1 
原来如此，其实我也有过这样的经历，话说2021年7月23日，我写了一篇[树](https://yuanfan.rbind.io/posts/tree/)，最后提到“狗主认真决定停笔一个月，好好沉淀一下”，哈哈，然而才过了2天，我就又写了一篇[食言而肥](https://yuanfan.rbind.io/story/eat-promise-will-be-fat/)。

至于说对自己写的东西的态度嘛，我感觉有些内容记录下来后过几年再看也还是会温暖我自己，有些思考过的东西好像又忘记，有些吐槽好傻气，不过那些 R 相关的过几年再看还是觉得很不太行呐，会感觉自己仍然有很大的进步空间，小说那块也是。说来说去，能够看到过去的自己写下的东西，总会给我带来一种不知道什么被填补了的感觉，不管以前是怎样，至少不是模糊的了。

**Reply by residualsun1** [reply]
> “为了不长胖，只好让阿木来写了……”

哈哈，“食言而肥”里最后一句让人忍俊不禁😆（论如何掩护打脸，阿木说这锅不背）

自己想的和别人看的有时候差异还挺大，比如你觉得不大行的，但读者却喜欢看。不过话说回来嘞，或许不一定说得上就是“为自己而写”，但相信也不是“为读者而写”，如果来客恰好有所感受甚至共鸣，那也会很开心。

> 说来说去，能够看到过去的自己写下的东西，总会给我带来一种不知道什么被填补了的感觉，不管以前是怎样，至少不是模糊的了。

达成共识🙌。

**Reply by CyrusYip** [reply]
> 之前见过叶寻提到自己的写作习惯是慢慢修改再发布，但我后来有了别的想法，觉得自己更喜欢抓住当下的情绪，然后就记录下来。
>
> ——@residualsun1

情绪确实稍纵即逝，不马上写下次就没那个感觉了。我还有一种情况：有强烈的情绪不吐不快，写了一篇之后没马上发，后面再看又感觉没那种强烈的情绪了，又不太想发。还有就是担心负面情绪转递给读者了。

我最近在考虑写文章不要太追求完美，降低一点标准。太追求完美导致我笔记软件里的草稿越来越多，现在感觉还是先发出来比较好，后续可以继续改。

**Reply by earfanfan** [reply]
>还有就是担心负面情绪转递给读者了。

我也有这种担心，以前沉浸在负面情绪中不知道如何处理的时候，我会尝试找朋友倾诉，后来自己渐渐有能力处理就都自己处理了。实在憋不住了，也会写一篇博客，其实仔细想想也许读者在看的时候因为“不在局中”，所以能够冷静看待，不会那么容易被影响的。

**Reply by residualsun1** [reply]
@CyrusYip 

> 强烈的情绪不吐不快，写了一篇之后没马上发，后面再看又感觉没那种强烈的情绪了，又不太想发。

我的第一反应是觉得你太过理性，而不是完美主义，觉得还是前者比后者更契合你。

> 还有就是担心负面情绪转递给读者了。

我没想过这一点，但觉得自己无论正面或负面情绪的文章都只会让读者觉得我太矫情，而我又恰好难以接受自己是个矫情的人……但事实证明我确实是，一时之间还改不了，那我想暂时就不为难自己了（大多数时候也是自己内耗的伤害最大）。如果人真的总是会变，看看以后会如何吧。

**Reply by earfanfan** [reply]
@residualsun1 
我没觉得你是矫情，相反，我是觉得原来男孩子的心思也可以这么细腻又敏锐呀，简直打破了我的刻板印象。其实“矫情”这个词可以看做是家庭和社会强行赋予我们的，就是要我们产生情绪耻感，只做他们希望我们做的事情。把“矫情”这个词拆开来看，为撒要轻易地批判“情”呢。

我想了下，叶寻同学的追求完美也可能是来源于一种压力，比照我自己的经历的话，曾有过类似的阶段。就是最开始的时候我的博客每个月浏览量才几百，那时候我就很好意思记录自己的情绪呀思考呀，后来有一天发现月浏览量超过一千了，就自然而然产生了一种想要“端庄”的困惑，大概就是更想要表现出自己成熟完美的一面，这种心理有点类似于“偶像包袱”之类的心理负担，其实我现在也还是会有点受影响的。记得之前在 COS 坛子里见过叶寻同学说自己博客有个月浏览量有9千，我猜这样的话，压力肯定比我大多了。

**Reply by residualsun1** [reply]
@earfanfan 很高兴能被你以自己的角度“看见”，这对我来说是一种很有力的肯定OwO

> 我想了下，叶寻同学的追求完美也可能是来源于一种压力，比照我自己的经历的话，曾有过类似的阶段。

我理解了，想起叶寻在博客里谈过Stack Overflow上的问答[经历](https://cyrusyip.org/zh-cn/posts/2021/05/14/why-accept-myself/)，我想追求完美在某种程度上也和“有益的点赞”是相近的，督促自己保持学习和进步。不过，我也有过好几次写了以后又不想发的经历，像叶寻那样，写的时候还是有情绪的，但是再看却感觉又缺了什么，就不想发了◑﹏◐。

**Reply by CyrusYip** [reply]
https://github.com/earfanfan/yuanfan/discussions/636#discussioncomment-10567172

我觉得你说的心理负担更多是源于自身，而不是具体某些读者给你造成压力，我看你博客的评论还是挺友善的。我好像没在统计之都说过我博客的访问量，不过你猜对了，最近三十天的访问量是九千左右，~~是时候加点广告了~~。

我最近几天在思考发文少的原因，我觉得主要是自己要求太高了。具体内容写成文章了：[少囤草稿，尽快发布 | 叶寻的博客](http://localhost:1313/zh-cn/posts/2024/09/19/publish-faster/)。

**Reply by earfanfan** [reply]
>我觉得你说的心理负担更多是源于自身，而不是具体某些读者给你造成压力，我看你博客的评论还是挺友善的。

其实从我的视角看，我表达的就是心理负担源于自身，并没有想要表达具体某些读者给我造成压力。

>我好像没在统计之都说过我博客的访问量，不过你猜对了，最近三十天的访问量是九千左右

啊，我伤心了。你这等于说我明明是猜的，却偏偏要说谎是在论坛看到的，我还以为咱们互为博主和读者这么多年，这点信任还是有的，没想到你第一反应还是怀疑我了，伤心了伤心了……

**Reply by CyrusYip** [reply]
@earfanfan 我记得没在任何地方说过博客访问量，于是去统计之都看[我最近十多条回复](https://d.cosx.org/u/CyrusYip)，也没有。然后就写下那条回复了。

> 你这等于说我明明是猜的，却偏偏要说谎是在论坛看到的

我想表达的是你可能记错了，而不是你在说谎。如果把「不过你猜对了」改为「不过你说的对」，这样会比较好吗？

**Reply by earfanfan** [reply]
@CyrusYip 
嗯，我想了想，即便是本就互相信任的朋友也会有误会的，毕竟语言文字确实天然具有模糊的特性，好啦，我不伤心了，我好了。

把“不过你猜对了”改成“不过你说的对”好像效果也差不多，因为分歧的根源是咱俩的记忆不一样，很大概率是你说撒我都会误会的。不过产生误会没关系的，解除就好了。PS我也记不清撒时候看到的了。

又PS，以前也有过类似的误会，我的记忆是在坛子里看到列弛留言了一句百年孤独全篇开头那样的文字，大约是“一个男孩很多年后仍然记得父亲带他去看冰块的那个下午”，然而列弛一点印象都冇得。
